pennylane/interfaces/execution.py

```diff
@@ -44,7 +44,17 @@
 
 device_type = Union[qml.Device, "qml.devices.Device"]
 
-jpc_interfaces = {"autograd", "numpy", "torch", "pytorch", "jax", "jax-python", "jax-jit"}
+jpc_interfaces = {

```

Comment:
is tensorflow-autograph the only one left?

Reply:
Yep, though it will also be the worst. I'm starting to feel a bit more confident about figuring it out, but it might come down to whether or not it will be worth the effort.